apps/openchallenges/challenge-service/src/main/resources/db/edam_concept.csv

```diff
@@ -0,0 +1,3474 @@
+"id","class_id","preferred_label"
```

Comment:
Can we additionally grab the `Parents` column?  IIRC, we had a discussion about building an ontology-like search in the OC, so getting the parent EDAM classes would be required.

Reply:
Sounds good - another item for our backlog! 😅